src/user/related_models/author_model.py

```diff
@@ -9,6 +9,7 @@
 from paper.utils import PAPER_SCORE_Q_ANNOTATION
 from purchase.related_models.purchase_model import Purchase
 from researchhub_case.constants.case_constants import APPROVED
+from user.related_models.author_institution import AuthorInstitution
```

Comment:
Nit: Is that change needed?

Reply:
Yes, it is needed because there is no `_init.py` file in that directory which includes the model. Django is unable to "see" it. I could alternatively create an _init file but didn't feel it was needed